### ISSUE TITLE
0.3.1 release

### ISSUE THREAD

**Issue by dougmet** [open-sdg/open-sdg#78]
The fix for #64 is relevant for all open-sdg users. Is it worth doing a `0.3.1` release? I've hooked up the UK site to the current head by referencing the commit. But a release tag is probably warranted.

**Comment by brockfanning** [comment]
Sounds good, though we'll want to update the changelog first, and make very clear that it's a breaking change for any sites that are already overriding a layout. If it's not too wordy, maybe the changelog should also describe how to "update" any overridden layouts (add the multilingual.html at the top, basically?) Or if that's too wordy maybe it could link to a Github issue describing it?

**Comment by dougmet** [comment]
Yes all sounds fine, I'll start working that up

**Comment by dougmet** [comment]
Fixed by #79
